frameworks/H2OAutoML/setup.sh

```diff
@@ -4,7 +4,7 @@ H2O_REPO=${2:-"https://h2o-release.s3.amazonaws.com/h2o"}
 echo "setting up H2O version $VERSION"
 
 HERE=$(dirname "$0")
-. ${HERE}/../shared/setup.sh
+. ${HERE}/../shared/setup.sh ${HERE}/../../
```

Comment:
H2O doesn't need it's own venv, so thanks for not changing this, this could (would) actually break the integration.